code/development-env/docker/docker-compose-app.yml

```diff
@@ -30,7 +30,7 @@ services:
     command:
       - cd /usr/src/app && yarn start
     volumes:
-      - $PWD/../../infrastructure-api:/usr/src/app/:ro
+      - $PWD/../../infrastructure-api:/usr/src/app/
```

Comment:
Think we should keep this read only, there is a switch we can add to prevent caching. I have it the flag into PR 330.